### DISCUSSION TITLE
Offer to support the use of databases

### DISCUSSION THREAD

**Discussion by agent188** [Rnen/AdminToolbox#114] [Ideas]
I suggest that you additionally be able to save player statistics in a database (MySQL, MariaDB, etc.).
Also the ability to transfer.

**Comment by Rnen** [comment]
It's a great idea really, i even thought about it a while ago
Problem is i know absolutely nothing on how to use a database with C# 😁

**Comment by SaengerItsWar** [comment]
here is a little example for mysql in c#
 https://git.purple-horizon-clan.net/SaengerItsWar/Ragemp-roleplay-skript/src/master/Database/PlayerDatabase.cs#L16

**Comment by Rnen** [comment]
Thanks for the example! I'll make sure to look into it as soon as i get some spare time :grin: 
Right now MegaPatch2 and real life is stealing all my time
